### ISSUE TITLE
bug: Invalid JSON logged in Docker console

### ISSUE THREAD

**Issue by mattdeluco** [localstack/localstack#5346]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The JSON strings output in the Docker console logs are not valid JSON.

Two issues I've noticed are single quotes, rather than double quotes, and `None` which is not a valid JSON type (I believe it should be `null`.)

```
24T17:27:59.815:WARNING:localstack.utils.cloudformation.template_deployer: Unexpected resource type ElasticBeanstalk::Environment when resolving references of resource Environment: {'Type': 'AWS::ElasticBeanstalk::Environment', 'Properties': {'ApplicationName': {'Ref': 'Application'}, 'EnvironmentName': {'Fn::Join': ['-', [{'Ref': 'EnvironmentName'}, {'Ref': 'DeveloperName'}]]}, 'Description': 'Default Environment', 'VersionLabel': 'Initial Version', 'TemplateName': 'DefaultConfiguration', 'OptionSettings': [{'Namespace': 'aws:elasticbeanstalk:application:environment', 'OptionName': 'AMM_S3_UPLOAD_BUCKET', 'Value': {'Ref': 'AppBucket'}}, {'Namespace': 'aws:elasticbeanstalk:application:environment', 'OptionName': 'AMM_TRANSACTIONUPLOAD_TOPIC', 'Value': {'Ref': 'TransactionUploadTopic'}}, {'Namespace': 'aws:elasticbeanstalk:application:environment', 'OptionName': 'AMM_TRANSACTIONUPLOAD_QUEUE', 'Value': {'Ref': 'TransactionUploadQueue'}}, {'Namespace': 'aws:elasticbeanstalk:application:environment', 'OptionName': 'AMM_TRANSACTIONUPLOAD_ROLE', 'Value': {'Ref': 'TransactionUploadRoleArn'}}, {'Namespace': 'aws:elasticbeanstalk:application:environment', 'OptionName': 'AMM_S3_UPLOAD_PREFIX', 'Value': 'uploads'}]}, 'LogicalResourceId': 'Environment', 'PhysicalResourceId': None}
```

### Expected Behavior

Assuming the intention here is to log JSON, I'd expect it to be valid. It would be far more convenient if it were valid because it would allows tools like `jq` to be used without intervention (fixing the broken JSON.)

```
24T17:27:59.815:WARNING:localstack.utils.cloudformation.template_deployer: Unexpected resource type ElasticBeanstalk::Environment when resolving references of resource Environment: {"Type": "AWS::ElasticBeanstalk::Environment", "Properties": {"ApplicationName": {"Ref": "Application"}, "EnvironmentName": {"Fn::Join": ["-", [{"Ref": "EnvironmentName"}, {"Ref": "DeveloperName"}]]}, "Description": "Default Environment", "VersionLabel": "Initial Version", "TemplateName": "DefaultConfiguration", "OptionSettings": [{"Namespace": "aws:elasticbeanstalk:application:environment", "OptionName": "AMM_S3_UPLOAD_BUCKET", "Value": {"Ref": "AppBucket"}}, {"Namespace": "aws:elasticbeanstalk:application:environment", "OptionName": "AMM_TRANSACTIONUPLOAD_TOPIC", "Value": {"Ref": "TransactionUploadTopic"}}, {"Namespace": "aws:elasticbeanstalk:application:environment", "OptionName": "AMM_TRANSACTIONUPLOAD_QUEUE", "Value": {"Ref": "TransactionUploadQueue"}}, {"Namespace": "aws:elasticbeanstalk:application:environment", "OptionName": "AMM_TRANSACTIONUPLOAD_ROLE", "Value": {"Ref": "TransactionUploadRoleArn"}}, {"Namespace": "aws:elasticbeanstalk:application:environment", "OptionName": "AMM_S3_UPLOAD_PREFIX", "Value": "uploads"}]}, "LogicalResourceId": "Environment", "PhysicalResourceId": null}
```

### How are you starting LocalStack?

With the `localstack` script

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    localstack start -d

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

Run a command that produces JSON logs, like `awslocal cloudformation create-stack`

### Environment

```markdown
- OS: macOS 11.6.2
- LocalStack: 0.13.3
```

**Comment by whummer** [comment]
Thanks for reporting @mattdeluco  - we have pushed a small fix in feafd0d which should dump the resources as JSON string (as opposed to serialized Python `dict`s). The `latest` Docker image is currently being built in our CI environment and will become available on Docker Hub in ~1h. Thanks

**Comment by mattdeluco** [comment]
Amazing! Thank you so much!
